ampbench_cli

```diff
@@ -205,10 +249,11 @@ const make_api_sd_validation = (api_validate_sd_return) => {
     };
 };
 
+
 program
-    .arguments('<url>')
-    .action(function(url) {
-        console.log('Analyzing url: %s', url);
+    .arguments('<url> <cssAddr>')
+    .action(function(url, cssAddr) {
+        cssValidator(cssAddr);
         analyze(url);
```

Comment:
Or should we give critical and people check on http://ampbench.appspot.com/api2?url= ?